apps/pwabuilder/src/script/pages/image-generator.ts

```diff
@@ -86,11 +86,6 @@ export class ImageGenerator extends LitElement {
           font-size: 10px;
         }
 
-
-        app-file-input:hover {
```

Comment:
because of the div element it has full width so the cursor pointer is misleading
![image](https://github.com/user-attachments/assets/705dc2db-343e-44b4-9257-85b6b36d7532)

the sl-button **already** has cursor pointer.